clidocstool_test.go

```diff
@@ -201,15 +203,38 @@ func TestGenAllTree(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, c.GenAllTree())
 
-	for _, tt := range []string{"buildx.md", "buildx_build.md", "buildx_stop.md", "docker_buildx.yaml", "docker_buildx_build.yaml", "docker_buildx_install.yaml", "docker_buildx_stop.yaml"} {
-		tt := tt
-		t.Run(tt, func(t *testing.T) {
-			bres, err := os.ReadFile(filepath.Join(tmpdir, tt))
+	seen := make(map[string]struct{})
+
+	require.NoError(t, filepath.Walk("fixtures", func(path string, info fs.FileInfo, err error) error {
```

Comment:
Can we use `filepath.WalkDir` for these? Not sure if we need the added features of `Walk` 🤔 


I'm wondering though if we're over-complicating things here; are we expecting to recurse here? Could we use a `filepath.Glob(path/*.md)` to get the list of markdown files? https://pkg.go.dev/path/filepath#Glob